scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/TypeProvider.scala

```diff
@@ -407,7 +407,10 @@ private[types] object TypeProvider {
   }
 
   private def formatString(xs: List[Any]): String =
-    xs.head.asInstanceOf[String].format(xs.tail: _*)
+    xs.tail match {
```

Comment:
@mrkm4ntr Thanks for this! wdyt about having it like 

```scala
xs match {
  case (head: String) :: Nil => head
  case (head: String) :: tail => head.format(xs.tail: _*)
  case _ => ""
}
```

Reply:
@regadas Thanks, fixed.

Reply:
@mrkm4ntr Thanks!